defaults/main.yml

```diff
@@ -2,9 +2,10 @@
 # WARNING: This may not be your desired MGMT address.
 nova_my_ip: '{{ ansible_default_ipv4.address }}'
 # RabbitMQ
-rabbit_hostname: 'localhost'
-rabbit_username: 'rabbit_username_default'
-rabbit_password: 'rabbit_pass_default'
+openstack_nova_compute_rabbit_hostname: 'localhost'
+openstack_nova_compute_rabbit_username: 'rabbit_username_default'
+openstack_nova_compute_rabbit_password: 'rabbit_pass_default'
```

Comment:
This would require changes across all roles, it's a good security measure to enforce it, but then the role is not installable without providing any variable. 
We could use random filter to generate a password in case in the it's not provided.
See:
http://docs.ansible.com/ansible/playbooks_lookups.html#the-password-lookup